src/main/java/com/mraof/minestuck/command/GetRungCommand.java

```diff
@@ -0,0 +1,34 @@
+package com.mraof.minestuck.command;
+
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.context.CommandContextBuilder;
+import com.mraof.minestuck.player.PlayerSavedData;
+import net.minecraft.commands.CommandSourceStack;
+import net.minecraft.commands.Commands;
+import net.minecraft.commands.arguments.EntityArgument;
+import net.minecraft.network.chat.Component;
+import net.minecraft.server.level.ServerPlayer;
+
+import static org.stringtemplate.v4.compiler.STLexer.str;
+
+public class GetRungCommand
+{
+	public static final String SUCCESS = "commands.minestuck.get_rung";
+	
+	public static void register(CommandDispatcher<CommandSourceStack> dispatcher)
+	{
+		dispatcher.register(Commands.literal("rung").then(Commands.argument("target", EntityArgument.player())
```

Comment:
The new `/rung` command is missing get and set subcommands. Currently it's:
```
/rung <player> //get
/rung <player> <rung> [progress] //set
```
With subcommands, it should be
```
/rung get <player> //get
/rung set <player> <rung> [progress] //set
```
or
```
/rung <player> get //get
/rung <player> set <rung> [progress] //set
```